pydoctor/epydoc/markup/__init__.py

```diff
@@ -108,16 +110,15 @@ def html2stan(html):
     stan.tagName = ''
     return stan
 
-def flatten(stan):
+def flatten(stan: Any) -> str:
```

Comment:
```suggestion
def flatten(stan: object) -> str:
```

Reply:
Any is much wider than object

Reply:
The problem with `object` is that as soon as `twisted.web.template.flattenString()` receives annotations, mypy will flag an error when passing an `object` as the second argument. While `Any` is not very descriptive, it is less fragile.

Reply:
That's the advantage of using object here, you get notified of the type change straight away

Reply:
You might want to set https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-disallow-any-explicit too

Reply:
But it won't happen straight away: it will happen on some random PR, which then either has to include a fix for an unrelated issue or be put on hold until a fix has been merged.

Reply:
pydoctor is almost 10k lines of code and very little of it has type annotations yet. The first priority is to get mypy coverage up to a level where it can detect inconsistencies. Making the ruleset  stricter is a good idea for the future, but doing that right would be counter-productive, in my opinion.

Reply:
I think I found a workable compromise: accept only `Tag` objects. That gives mypy a specific type to work with, while we can always wrap strings, sequences etc. in a transparent tag to convert them to a `Tag` object.